src/plugins/intel_npu/src/plugin/npuw/partitioning/partitioning.cpp

```diff
@@ -202,7 +202,8 @@ class Partitioner {
     void createFunction(FunctionPipeline& func_ggg);
 
     template <typename T, typename M>
-    void rearrange_to_function_protocol(const std::vector<T>& protocol, std::vector<T>& call, const M& call_to_proto) {
+    void rearrange_to_function_protocol(const std::vector<T>& protocol, std::vector<T>& call, const M& call_to_proto,
+                                        const std::shared_ptr<ov::Model>& model) {
```

Comment:
to avoid confusion, I'd rename here `model` to `fcall` (standing for function call)

Reply:
Used another variable, but thank you!